### PR TITLE
Update doc to mention available installation via brew

### PR DESCRIPTION
**Pull request by florentferry**
Update README, and documentation to mention available installation via brew for OS X users. #2

### CODE REVIEW

**Comment by k4nar** [comment]
cc @jmaupetit

File: docs/index.md [hunk 1]
```diff
@@ -33,6 +33,12 @@ $ pip install td-watson
 
 Depending on your system, you might need to run this command with root privileges in order to install Watson globally.
 
+On OS X, the easiest way to install Watson is using [Homebrew](http://brew.sh/):
+
+```bash
+  $ brew update && brew install watson
```
Comment (jmaupetit) [inline docs/index.md]: Indentation is not required here 😸

**Comment by jmaupetit** [comment]
Thank you for this @florentferry! After a small fix, this PR will be merged.

**Comment by florentferry** [comment]
@jmaupetit, I submitted the fix about indentation.

**Pull request merged**